model/src/channel/mod.rs

```diff
@@ -191,23 +191,32 @@ impl<'de> Visitor<'de> for GuildChannelVisitor {
         const VARIANTS: &[&str] = &[
             "GuildCategory",
             "GuildNews",
+            "GuildNewsThread",
+            "GuildPrivateThread",
+            "GuildPublicThread",
             "GuildStore",
             "GuildText",
             "GuildVoice",
         ];
 
         let mut bitrate = None;
+        let mut default_auto_archive_duration: Option<Option<AutoArchiveDuration>> = None;
```

Comment:
Is this a use case for `NullableField`?

Reply:
No, those are used during serializing of fields we send to Discord. Clippy doesn't actually complain about this `≤Option<Option<>>`.

Reply:
ah ok. just wanted to check that abstraction couldn't be used elsewhere too.